compiler/src/iree/compiler/Dialect/HAL/Target/CUDA/BUILD

```diff
@@ -1,49 +0,0 @@
-# Copyright 2021 The IREE Authors
-#
-# Licensed under the Apache License v2.0 with LLVM Exceptions.
-# See https://llvm.org/LICENSE.txt for license information.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
-package(
-    default_visibility = ["//visibility:public"],
-    features = ["layering_check"],
-    licenses = ["notice"],  # Apache 2.0
-)
-
-# Temporarily disabled pending build system changes.
-# cc_library(
-#     name = "CUDA",
```

Comment:
We could remove the commented out lines from here too: https://github.com/iree-org/iree/blob/a8e4c38c8f7ef02c2ed71fdd4b4f5312fcced0f9/compiler/src/iree/compiler/Tools/BUILD#L147-L167 

(or keep them, to more directly show that CUDA is not included there, I'm fine with either)

Reply:
good point, done